snapcraft.yaml

```diff
@@ -1447,8 +1447,10 @@ parts:
       fi
```

Comment:
Hmm, seeing `CRAFT_STAGE` here feels wrong as we are in the build step.

Reply:
I believe `CRAFT_STAGE` is being used here because there is a check in `strip:override-prime` which primes the documentation only if `arch != riscv64`. I think this check can be moved to the `lxd` build step though, and then we can just prime the documentation there. I'm still learning about snap though, so I may be off here :)

Reply:
Sorry for not closing the loop here, I dealt with it in https://github.com/canonical/lxd-pkg-snap/pull/545